runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java

```diff
@@ -214,7 +214,7 @@
 		BLACK_MASK, BLACK_MASK_I, BLACK_MASK_1, BLACK_MASK_1_I, BLACK_MASK_2, BLACK_MASK_2_I, BLACK_MASK_3, BLACK_MASK_3_I, BLACK_MASK_4, BLACK_MASK_4_I, BLACK_MASK_5,
 		BLACK_MASK_5_I, BLACK_MASK_6, BLACK_MASK_6_I, BLACK_MASK_7, BLACK_MASK_7_I, BLACK_MASK_8, BLACK_MASK_8_I, BLACK_MASK_9, BLACK_MASK_9_I, BLACK_MASK_10_I,
 		SLAYER_HELMET, SLAYER_HELMET_I, BLACK_SLAYER_HELMET, BLACK_SLAYER_HELMET_I, PURPLE_SLAYER_HELMET, PURPLE_SLAYER_HELMET_I, RED_SLAYER_HELMET, RED_SLAYER_HELMET_I,
-		GREEN_SLAYER_HELMET, GREEN_SLAYER_HELMET_I, TURQUOISE_SLAYER_HELMET, TURQUOISE_SLAYER_HELMET_I, HYDRA_SLAYER_HELMET, HYDRA_SLAYER_HELMET_I),
+		GREEN_SLAYER_HELMET, GREEN_SLAYER_HELMET_I, TURQUOISE_SLAYER_HELMET, TURQUOISE_SLAYER_HELMET_I, TWISTED_SLAYER_HELMET, TWISTED_SLAYER_HELMET_I, TWISTED_HORNS, HYDRA_SLAYER_HELMET, HYDRA_SLAYER_HELMET_I),
```

Comment:
The Horns should be in their own mapping; How you've got it written now means the Twisted Horns use the value of the Black Mask.

Look at how the Bonecrusher Necklace is mapped (lines 230 and 231)

Reply:
Thanks for the review. Do you mean like this:

ITEM_TWISTED_HORNS(TWISTED_HORNS, SLAYER_HELMET, SLAYER_HELMET_I),

Reply:
That looks correct. 
The helmets should be kept in the Black Mask list as well, since they still break down into that.

Reply:
Will do, thank you